src/shared/modules/dbMeta/dbMetaDuck.js

```diff
@@ -205,6 +206,7 @@ const initialState = {
   databases: [],
   settings: {
     'browser.allow_outgoing_connections': false,
+    'browser.remote_content_hostname_allowlist': null,
```

Comment:
Done this way so we get a falsy value when it's not set. Otherwise it will never read the whitelist

Reply:
wdyt about just having `browser.remote_content_hostname_allowlist` locally cached?
Browser front-end reads from single source (via selector `getRemoteContentHostnameAllowlist`), but both `browser.remote_content_hostname_allowlist` and `browser.remote_content_hostname_allowlist` from server responses write to that local cache?

Reply:
I've added this and a new test to check that it works now :)